### DISCUSSION TITLE
Installation issues

### DISCUSSION THREAD

**Discussion by RBSJonker** [LCA-ActivityBrowser/activity-browser#1247] [Q&A] [answered]
Following the installation of AB (Activity Browser), attempting to access the activity browser results in the following error message:

'activity-browser' is not recognized as an internal or external command, operable program or batch file.

Additional Information:

Anaconda is installed on the admin account of my laptop.
The issue occurs while working on my study account.
Could this problem be related to permissions, requiring approval from the admin account?
Seeking assistance to resolve this matter. Any insights or guidance would be greatly appreciated!

**Comment by marc-vdm** [comment]
Did you first `conda activate ab` to activate the correct environment in the anaconda terminal? This is required for the command to be recognized.

**Reply by RBSJonker** [reply]
Hi Marc,

Yes i did that

**Reply by marc-vdm** [reply]
Could you share a screenshot of your terminal by any chance? That may help us in understanding your problem

**Reply by RBSJonker** [reply, marked as answer]
Hi Marc, thanks for you quick response!
I found the solution! 

1) Reintalling Anaconda on the windows account I am using
2) Reinstalling AB!

**Reply by marc-vdm** [reply]
Happy we figured it out so soon :)
